test/test_matmul_scheduler.cpp

```diff
@@ -406,24 +408,1801 @@ TEST_F(MatmulSchedulerTest, EpilogueAlphaOutputCast_CUDA) {
   NVF_CHECK(outputs[0].allclose(tref, 0.001, 0.001));
 }
 
-// Matmul test that relies on segmenter for 'C = relu(A x B)' fusion, for
-//  Ampere
-TEST_F(MatmulSchedulerTest, EpilogueRelu_CUDA) {
+// Matmul test that uses segmenter for 'D = relu(A x B)' fusion,
+//  Precisions:
+//   - A (fp16 - h)
+//   - B (fp16 - h)
+//   - math (fp32 - s)
+//   - D (fp32 - s)
+//  Target architectures: Turing, Ampere
+TEST_F(MatmulSchedulerTest, EpilogueReluHSS_CUDA) {
```

Comment:
remove the `_CUDA` suffix, same for all tests

Reply:
Good point - will remove the `_CUDA` suffix with the upcoming update of this MR.

Thanks!